tsconfig.json

```diff
@@ -13,7 +13,8 @@
         "esModuleInterop": true,
         "resolveJsonModule": true,
         "isolatedModules": true,
-        "typeRoots": ["node_modules/@types", "types"]
+        "typeRoots": ["node_modules/@types", "types"],
+        "skipLibCheck": true
```

Comment:
Do we need `skipLibCheck`?

Reply:
Possibly not, made a small mistake here with using `npm` instead of `yarn`, so some transient deps may be using the wrong versions. I was getting errors running a build from incorrect types in a `@types/...` package. Will switch to yarn and see if it's still needed.